proto/cosmos/gov/v1beta2/gov.proto

```diff
@@ -121,3 +121,9 @@ message TallyParams {
   //  vetoed. Default value: 1/3.
   string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec"];
 }
+
+// Content is used to wrap the legacy content field into a message. This ensures backwards compatibility
+message MsgContent {
+  google.protobuf.Any content   = 1 [(cosmos_proto.accepts_interface) = "Content"];
+  string              authority = 2;
+}
```

Comment:
MsgContent is a clever trick so that we can work with one MsgSubmitProposal in v1beta2. I'm wondering though if it'll not bite us later, because MsgContent is not a real `service Msg{rpc MsgContent}`

What happened to the `v1beta2.MsgSubmitLegacyProposal` idea? I think for explicitness, I prefer it over MsgContent.

Reply:
Would you prefer that we rename `v1beta2.MsgContent` to `v1beta2.MsgSubmitLegacyProposal`. I'm not sure if `MsgContent` is something even exposed to clients

Reply:
Yeah, I would rename it, move it to `tx.proto` and also add a `rpc MsgSubmitLegacyProposal`, and we would also need a `MsgSubmitLegacyProposalResponse`

Reply:
But it never gets executed. Why add it to the msg server?

Reply:
OK let's keep it like this. My concern is that clients see this proto def and get confused as to why this Msg is not like other Msgs.. I'll add a proto comment about this to clarify.

Also, what do you think of these ideas:
1. rename to v1beta2.MsgLegacyContent ?
2. call it something that doesn't start with `Msg`, like `LegacyContent`
3. move it into a new file `internal.proto`, so that (maybe) clients will know it's internal and they should never use it
4. move to v1beta1.MsgContent ?

Probably the proto comment is enough.

Reply:
Moving forward, would we need to add MessageOptions (required_signers, #10933, then validate_basic) to this MsgContent. @aaronc any thoughts on this?

Reply:
Yeah I think `MsgLegacyContent` is probably more informative so we can rename it to that. I also don't mind dropping the `Msg` prefix. The reason why it is there is that it is, after all, a `sdk.Msg` and we need to be able to convert the legacy `Content` type into a `sdk.Msg` type to be compatible with the new proposal structure